src/edu/gatech/cs2340/trydent/Time.java

```diff
@@ -0,0 +1,103 @@
+package edu.gatech.cs2340.trydent;
+
+/**
+ * Static class to keep track of time.
+ * 
+ * All values returned by public methods are in seconds,
+ * unless stated otherwise.
+ * 
+ * @author Garrett Malmquist
+ */
+public class Time {
+
+	// All private variables are stored in milliseconds for accuracy.
+	private static volatile double gameTimeSinceStartup;
+	private static volatile long realTimeSinceStartup;
+	private static volatile long realLastFrame;
+	private static volatile long realStartTime;
+	private static volatile double gameTimePassed;
+	private static volatile long realTimePassed;
+	private static volatile double timeRate;
+	
+	/**
+	 * Game time since the game started.
+	 * @return
+	 */
+	public static double getTime() {
+		return gameTimeSinceStartup / 1000.0;
```

Comment:
Mmmmm I guess it wouldn't hurt. I'm sure the compiler will optimize it away if I make it private static final.